src/pybind/mgr/osd_perf_query/module.py

```diff
@@ -97,22 +99,34 @@ def handle_command(self, inbuf, cmd):
                 column_names = ["client_id"]
             for d in descriptors:
                 desc = d
-                if d in ['write_bytes', 'read_bytes']:
+                if d in ['bytes']:
```

Comment:
Nit: it's not immediately obvious to me why ```bytes``` gets special treatment over ```write_bytes``` and ```read_bytes``` here and below?

Reply:
I had to add 'bytes' to descriptors, because I wanted to order by this counter in the limit. I didn't want to output it in the table though, because it would look redundant and a little strange (if we output total bytes, why then not total ops and latency?). Anyway, as it is only an example I open to any suggestions.